src/wallet/rpc/backup.cpp

```diff
@@ -1784,25 +1784,40 @@ RPCHelpMan listdescriptors()
 
     LOCK(wallet->cs_wallet);
 
-    UniValue descriptors(UniValue::VARR);
     const auto active_spk_mans = wallet->GetActiveScriptPubKeyMans();
+
+    std::vector<std::tuple<std::string, WalletDescriptor, bool, std::optional<bool>>> wallet_descriptors;
```

Comment:
nit: I find tuples a bit hard to grok. Could we just pass a comparator to `GetAllScriptPubKeyMans`?